src/main/kotlin/blackjack/domain/BlackJackGame.kt

```diff
@@ -12,8 +12,33 @@ class BlackJackGame(
                     BlackJackPlayResult.getResult(it.getBestSum(), dealer.getBestSum()),
                 )
             }
-        val winCount = playerResults.filter { it.result == BlackJackPlayResult.LOSE }.size
-        val loseCount = playerResults.filter { it.result == BlackJackPlayResult.WIN }.size
+        val winCount = playerResults.filter { it.result == BlackJackPlayResult.LOSE }.count()
```

Comment:
count는 아래 처럼 사용이 가능합니다! 

```suggestion
        val winCount = playerResults.count { it.result == BlackJackPlayResult.LOSE }
```